client/src/nonblocking/quic_client.rs

```diff
@@ -358,8 +353,12 @@ impl QuicClient {
             last_error, self.addr
         );
         // If we get here but last_error is None, then we have a logic error
-        // in this function, so panic here with an expect to help debugging
-        Err(last_error.expect("QuicClient::_send_buffer last_error.expect"))
```

Comment:
This one should explicitly not be changed

Reply:
Agreed, I think this should remain as is.

Reply:
reverted